main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	entities "main/entities"
+	handlers "main/handlers"
+	utils "main/utils"
+	"net/http"
+)
+
+func main() {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/books", handlers.HandleBookRequests)
+
+	go func() {
+		runTest()
+	}()
+
+	http.ListenAndServe(":8080", mux)
+}
+
+func runTest() {
```

Comment:
I run here some test to show how API works
First we get all the books with GET method
Later we add new book with PUSH method
And finally, get all the books again with GET to see the difference